backends/google/sheets/google-sheets.js

```diff
@@ -57,7 +57,7 @@ export default class GoogleSheets extends Google {
 			}
 
 			file.objectKeys = new Map(); // "string" => "string"
-			if (this.options.headerRow) {
+			if (this.options.headerRow === true) {
```

Comment:
Why?

Reply:
Usually JS APIs accept any truthy value, unless they want to make a distinction between booleans and other value types.

Reply:
Ah, OK. I'll revert to the previous version, then.